schemas/context/segmentdefinition-meta-base.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segment-meta-base",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segments basic metadata",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/segmentdefinition"],
+  "description": "Common fields required for any segment definition",
+  "definitions": {
+    "segment-meta-base": {
+      "properties": {
+        "xdm:entityIdentity": {
```

Comment:
Is this a segment identity or an entity identity? The title and property name would be less confusing if they were aligned.